src/odbc.cpp

```diff
@@ -167,20 +166,24 @@ void ODBCAsyncWorker::OnError(const Napi::Error &e) {
     ODBCError odbcError = errors[i];
     Napi::Object errorObject = Napi::Object::New(env);
 
-    errorObject.Set(
+    errorObject.Set
+    (
       Napi::String::New(env, STATE),
       #ifdef UNICODE
-      Napi::String::New(env, (const char16_t*)odbcError.state)
+      Napi::String::New(env, (odbcError.state != NULL) ? (const char16_t*)odbcError.state : (const char16_t*)"")
```

Comment:
This won't work because the `""` only has 1 NUL byte, so it will not find the terminator and flow past the end.

```suggestion
      Napi::String::New(env, (odbcError.state != NULL) ? (const char16_t*)odbcError.state : u"")
```

Reply:
If you can't guarantee C++11 support, then:

```suggestion
      Napi::String::New(env, (odbcError.state != NULL) ? (const char16_t*)odbcError.state : (const char16_t*)L"")
```